newsfragments/8626.feature.rst

```diff
@@ -0,0 +1,8 @@
+Add server option ``--proxy-trusted-addresses`` to enabled parsing of proxy headers from trusted addresses.
+
+.. important::
+  Breaking change: Previous the server used the default value provided by ``uvicorn`` which always enabled the proxy header for localhost proxy.
+  If you want to have the same behavior as before, you need to set the ``--proxy-trusted-addresses`` to ``localhost,127.0.0.1,::1``.
+
+.. tip::
+  The env variable ``PARSEC_PROXY_TRUSTED_ADDRESSES`` instead of the option.
```

Comment:
Better avoid fancy formatting and stick to oneliners for changelog (see below)
```suggestion
Add server option ``--proxy-trusted-addresses`` to enable parsing of proxy headers from trusted addresses.
```

Reply:
I'd like to have the migration steps in the changelog

Reply:
outdated